### ISSUE TITLE
Ventricular bigeminy

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7592]
**Preferred term label:**
Ventricular bigeminy

**Synonyms**

**Definition (free text, please give PubMed ID)**

An electrocardiographic finding of a normal QRS followed by a premature ventricular contraction; a rhythmic pairing of normal and atypical beats originating in the ventricles in a 1-1 ratio such that an ectopic ventricular beat follows each regular heartbeat

PMID: [19043599](https://pubmed.ncbi.nlm.nih.gov/19043599)
**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 616249   PMID:23388215  1/5

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
